code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -76,6 +76,30 @@
 	melee_damage_lower=10
 	melee_damage_upper=40
 
+/mob/living/simple_animal/hostile/bear/spare
+	name = "spare bear"
+	desc = "This bear has adapted a form of camouflage from generations of natural selection in which the omnivores scavenge from space stations and their dumpsters. Its golden skin fools card scanners into opening the door."
+	health = 300
+	maxHealth = 300
+	melee_damage_lower = 15
+	melee_damage_upper = 35
+	icon_state = "sparebear"
+	icon_dead = "sparebear_dead"
+	default_icon_floor = "sparebear"
+	default_icon_space = "sparebear"
+
+/mob/living/simple_animal/hostile/bear/spare/getarmor(var/def_zone, var/type)
+	if(type == "laser")
+		return 80
+	return 10
+
+/mob/living/simple_animal/hostile/bear/spare/getarmorabsorb()
+	return 25
```

Comment:
Won't this, combined with the almost near immunity to lasers, make it far too tanky for anything that isn't a shotgun?

Reply:
yes, it's rather shiny and armored